### DISCUSSION TITLE
Change default configuration for computing workflows

### DISCUSSION THREAD

**Discussion by pkalbhor** [cms-AlCaDB/AlCaVal#26] [General]
Asked by @tvami. 
Current default configuration while creating a ticket=> Memory: 16 GB, Cores: 8
Update them to Memory: 8 GB, Cores: 4

**Comment by pkalbhor** [comment]
Would love to know compulsion behind this decision. Were AlCa consuming more than enough resources?

**Reply by tvami** [reply]
It's not for AlCa only, all RelVals are moving to this schema. The reason is that T0 uses the prev config so our jobs are competing with T0 doing Prompt reco. With the new config we could get better availability of machines.

**Reply by pkalbhor** [reply]
Thanks Tamas, we will change default configuration to 8GB/4 cores on alcaval with next update
